src/plinder/data/structure/models.py

```diff
@@ -50,23 +49,23 @@ class MonomerName(str, Enum):
     predicted = "predicted"
 
 
-class ExcludeComputedModelBase(BaseModel, metaclass=abc.ABCMeta):
-    def model_dump(self, **kwargs) -> dict[str, Any]:  # type: ignore
-        model_dict = super().model_dump(**kwargs)
-        model_fields = set(self.model_fields.keys())
-        return {key: value for key, value in model_dict.items() if key in model_fields}
-
-    # @model_serializer()
-    # def exclude_computed(self) -> dict[str, Any]:
-    #     model_dict = super().model_dump()
-    #     model_fields = set(self.model_fields.keys())
-    #     return {key: value for key, value in model_dict.items() if key in model_fields}
-
-
-def manual_model_dump(model_to_dump: ExcludeComputedModelBase) -> dict[str, Any]:
-    return model_to_dump.model_dump()
-
-
-ExcludeComputedModel = Annotated[
-    ExcludeComputedModelBase, PlainSerializer(manual_model_dump)
-]
+class DocBaseModel(BaseModel):
+    @classmethod
+    def get_docstrings(cls) -> list[tuple[str, str | None]]:
+        """
+        Returns a list of all properties in the class.
+
+        Returns:
+        --------
+        list
+            A list of tuples containing the name and docstring of each property.
+        """
+        attributes = [
+            (name, value.description) for name, value in cls.model_fields.items()
+        ]
+        others = [
+            (name, prop.__doc__)
+            for name, prop in cls.__dict__.items()
+            if isinstance(prop, cached_property) or isinstance(prop, property)
+        ]
+        return attributes + others
```

Comment:
Instead as a list of tuples, I think it is more idiomatic to return a dictionary instead, mapping the property/attribute names to their respective description.